toolkit/scripts/download_spec_sources.sh

```diff
@@ -21,7 +21,7 @@ trap clean_up EXIT SIGINT SIGTERM
 
 echo "Will download to '$TEMP_DIR':"
 
-OUTPUT_DIR="$(pwd)/downloaded_sources"
+OUTPUT_DIR="$HOME/Mariner_downloaded_sources"
```

Comment:
In my opinion, we should not populate a directory outside the project `root`.

Probably an alternative could be expecting the user to supply a directory where the downloaded files should be placed instead of having a default.

Reply:
Agree with Falak here. The only directory our project "owns" by default is our project root.
Could we give an option to the user instead to specify a different location than the default?

Reply:
Having a common location is useful when working with multiple copies of the repo, but agree a parameter would be a good compromise.

Reply:
I actually wanted to keep the script simple and avoid a mix of named arguments and a list of specs, for which the script downloads the sources. Especially since this is a simple internal help script, not something to expect complex and public use.

Considering that and following the idea of which directories can be viewed as "owned" by the project/script, what do you think about an alternative where we update `.gitignore` with `downloaded_sources/`?

Reply:
In my opinion, the `.gitignore` is a cleaner approach :)